webviz_examples/webviz-full-demo.yml

```diff
@@ -161,13 +159,12 @@ layout:
             content:
               - DiskUsage:
                   scratch_dir: ../.
-                  date: "2021-07-09"  # date only needed if requesting a specific report, otherwise using latest available.
+                  date: "2021-07-09" # date only needed if requesting a specific report, otherwise using latest available.
```

Comment:
I guess this is a VS Code plugin that changes other unrelated lines? Seems like that plugin and GitHub does not agree on number of spaces before comment 😄

Reply:
Format on save in VS Code seems to disagree with GitHub here. Fixed 👍